### PR TITLE
TMA-568: Users Brick: add support for other user's attributes

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/146/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 2s
- gooddata-ruby-rubocop-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED

**Comment by vhtien** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/147/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 1s
- gooddata-ruby-rubocop-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED

**Comment by vhtien** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/148/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 2s
- gooddata-ruby-rubocop-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED

**Comment by petr-klemsinsky** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/149/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 1s
- gooddata-ruby-rubocop-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED

File: lib/gooddata/lcm/actions/synchronize_users.rb [hunk 1]
```diff
@@ -124,9 +131,14 @@ def call(params)
...
+            :user_group => row[user_groups_column] && row[user_groups_column].split(',').map(&:strip),
             :pid => multiple_projects_column.nil? ? nil : (row[multiple_projects_column] || row[multiple_projects_column.to_sym]),
-            :language => row[language_column] || row[language_column.to_sym]
+            :language => row[language_column] || row[language_column.to_sym],
+            :company => row[company_column] || row[company_column.to_sym],
+            :position => row[position_column] || row[position_column.to_sym],
+            :country => row[country_column] || row[country_column.to_sym],
+            :phone => row[phone_column] || row[phone_column.to_sym],
+            :ip_whitelist => ip_whitelist
           }.compact
```
Comment (panjan) [inline lib/gooddata/lcm/actions/synchronize_users.rb]: Please write a unit test that checks the result of the action.

Comment (vhtien) [reply lib/gooddata/lcm/actions/synchronize_users.rb]: there: https://ms-gerrit.intgdc.com/#/c/740/

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/74/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1016/ : FAILURE in 18s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/161/ : SUCCESS in 23s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/899/ : SUCCESS in 45m 32s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/672/ : SUCCESS in 21m 00s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/75/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/1017/ : SUCCESS in 42s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/162/ : SUCCESS in 21s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/900/ : SUCCESS in 45m 05s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/673/ : SUCCESS in 23m 42s

**Pull request merged**